include/swift/SIL/SILCloner.h

```diff
@@ -1252,15 +1252,33 @@ void
 SILCloner<ImplClass>::visitWitnessMethodInst(WitnessMethodInst *Inst) {
   auto conformance =
     getOpConformance(Inst->getLookupType(), Inst->getConformance());
+  auto lookupType = Inst->getLookupType();
+  auto newLookupType = getOpASTType(lookupType);
+  SILValue OpenedExistential;
+  if (Inst->hasOperand())
+    OpenedExistential = getOpValue(Inst->getOperand());
+  else if (isa<ArchetypeType>(newLookupType) &&
+           !cast<ArchetypeType>(newLookupType)
+                ->getOpenedExistentialType()
+                .isNull()) {
```

Comment:
Isn't the test from 1260-1253 equivalent to newLookupType->isOpenedExistential()?